assets/queries/openAPI/link_object_incorrect_ref/query.rego

```diff
@@ -6,29 +6,14 @@ CxPolicy[result] {
 	doc := input.document[i]
 	openapi_lib.check_openapi(doc) != "undefined"
 
-	link := doc.components.responses[r].links[l]
-	openapi_lib.incorrect_ref(link["$ref"], "links")
+	[path, value] := walk(doc)
 
-	result := {
-		"documentId": doc.id,
-		"searchKey": sprintf("components.responses.{{%s}}.links.{{%s}}.$ref={{%s}}", [r, l, link["$ref"]]),
-		"issueType": "IncorrectValue",
-		"keyExpectedValue": "Link ref points to '#/components/links'",
-		"keyActualValue": "Link ref does not point to '#/components/links'",
-	}
-}
-
-CxPolicy[result] {
-	doc := input.document[i]
-	openapi_lib.check_openapi(doc) != "undefined"
-
-	link := doc.paths[path][operation].responses[r].links[l]
-	openapi_lib.content_allowed(operation, r)
-	openapi_lib.incorrect_ref(link["$ref"], "links")
+	ref := value.links[l]["$ref"]
```

Comment:
I think we should ignore `components.links`, since it can have an external reference...